git-flow.txt

```diff
@@ -0,0 +1,17 @@
+1. Buat branch baru untuk setiap fitur yang akan dibuat dan bug yang di-resolve, dapat juga hanya 1 branch sesuai dengan kebutuhan. Lakukan perintah berikut untuk membuat branch baru :
```

Comment:
please change to be real branch name, based on the example questions,
like release/xxx , feature/xxx, etc.